packages/web-components/src/components/chat/components/formulaElement/src/formulaElement.ts

```diff
@@ -33,6 +33,36 @@ export default class formulaElement extends LitElement {
   @state()
   formula;
 
+  /**
+   * startMathJax -  edit target document for rendering
+   */
+  async startMathJax() {
```

Comment:
`import 'mathjax/es5/tex-mml-chtml.js';` still pollute the global "window" object. This means that if the chat is running along side other code that uses mathjax (not a crazy thought) that they will clobber each other. Going to approve the PR, but we gotta work this out or move to katex.